utils/timer/clock_test.go

```diff
@@ -0,0 +1,37 @@
+package timer
+
+import (
+	"testing"
+	"time"
+)
+
+func TestClockSet(t *testing.T) {
+	clock := Clock{false, time.Unix(1000000, 0)}
```

Comment:
I think this would be clearer, and more in line with how Clock is actually used, if the struct creation used the zero value of the struct here

Reply:
Okay, I think commit 8f54a7c does what you're asking